### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by amashutikov**
[DEMO LINK](https://amashutikov.github.io/react_list-of-goods-hooks/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,49 +18,86 @@ export const goodsFromServer = [
...
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [isReversed, setDirection] = useState(false);
```
Comment (33Orange) [inline src/App.tsx:28]: Let's follow the naming convention for the useState. 
`const [value, setValue] = useState()`

```suggestion
  const [isReversed, setIsReversed] = useState(false);
```

File: src/App.tsx [hunk 2]
```diff
@@ -15,49 +18,86 @@ export const goodsFromServer = [
...
+  const [sortType, setSort] = useState(SortType.NONE);
```
Comment (33Orange) [inline src/App.tsx:29]: Same here 

```suggestion
  const [sortType, setSortType] = useState(SortType.NONE);
```

File: src/components/GoodsList.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+          <li
+            data-cy="Good"
+            key={good}
+          >
+            {good}
+          </li>
+        );
+      })}
+    </ul>
+  </ul>
```
Comment (33Orange) [inline src/components/GoodsList.tsx:19]: Why did you wrap it to `<ul>` element 2 times ?

**Review by 33Orange — CHANGES_REQUESTED** [review]
Almost done

A few moments to fix

File: src/components/GoodsList.tsx (hunk 3 above)
Comment (amashutikov) [reply src/components/GoodsList.tsx:19]: It was wrapped with two ul's in the original App component

File: src/components/GoodsList.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+type Props = {
+  goods: string[];
+};
+
+export const GoodsList: React.FC<Props> = ({ goods }) => (
+// It was wrapped with two ul's in the original App component
```
Comment (dinaminahi) [inline src/components/GoodsList.tsx:6]:
```suggestion
```

**Review by dinaminahi — APPROVED** [review]
Well done 👍

**Pull request closed without merging**